pages/episodes/[slug].vue

```diff
@@ -53,6 +53,21 @@ const isMobile = computed(() => {
             <div class="grid">
               <div class="col">
                 <div v-if="dataLoaded" class="episode flex">
+                  <Html>
+                    <Head>
+                      <Title>{{ episode.title }} | Radiolab | WNYC Studios</Title>
+                      <Meta name="description" :content="episode.tease" />
+                      <Meta name="og:title" :content="`${episode.title} | Radiolab | WNYC Studios`" />
+                      <Meta name="og:description" :content="episode.tease" />
+                      <Meta name="og:type" content="article" />
+                      <Meta name="og:image" :content="episode['image-main'].url" />
+                      <Meta name="og:image:width" :content="`${episode['image-main'].w}`" />
+                      <Meta name="og:image:height" :content="`${episode['image-main'].h}`" />
+                      <Meta name="twitter:title" :content="`${episode.title} | Radiolab | WNYC Studios`" />
+                      <Meta name="twitter:description" :content="episode.tease" />
+                      <Meta name="twitter:image" :content="episode['image-main'].url" />
```

Comment:
2:1 ration for Twitter

"episode['image-main'].template.replace('%s/%s/%s/%s', '1200/600/c/80')"